src/app/shared/theme-picker/theme-picker.spec.ts

```diff
@@ -15,7 +15,7 @@ describe('ThemePicker', () => {
     const component = fixture.componentInstance;
     const name = 'pink-bluegrey';
     spyOn(component.styleManager, 'setStyle');
-    component.installTheme(name);
+    component.onThemeSelection(name);
```

Comment:
I prefer the previous name; my philosophy is that method names should describe what they do, not how they're called